packages/provider/src/RelayClient.ts

```diff
@@ -103,6 +108,8 @@ export class RelayClient {
   config!: GSNConfig
   dependencies!: GSNDependencies
   private readonly rawConstructorInput: GSNUnresolvedConstructorInput
+  publicKey!: string
+  cipher!: deoxysii.AEAD
```

Comment:
you will be able to use `sapphire.cipher.X25519Deoxysii` once the runtime upgrade happens and the key generation algs are aligned.